com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -55,13 +57,15 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
         internal static bool UseClassicDelta = true;
         internal static bool UseSnapshot = false;
 
+        private const int k_TimeSyncFrequency = 30; // sync every 30 ticks, TODO will be removed once timesync is done via snapshots
```

Comment:
since tick length is configurable by users, shouldn't this const be in seconds? This way we have predictable behaviour no matter what users set?

Reply:
Fixed, syncing every second now regardless of tickrate.